main.rb

```diff
@@ -0,0 +1,29 @@
+require_relative 'app'
+
+my_class = App.new
+
+@list = ['1 - List all books', '2 - List all people', '3 - Create a person', '4 - Create a book',
+         '5 - Create a rental', '6 - List all rentals for a given person id', '7 - Exit']
+
+def method(my_class)
```

Comment:
[OPTIONAL]
Hi Mahdi,

Your code looks functional and seems to get the job done, which is great!

One thing that I'd like to point out is the naming of your method, which is currently called `method`. It's generally a good practice to give your methods intuitive and descriptive names so that anyone reading the code can easily understand what each method is supposed to do. Naming it `method` is quite generic and does not provide much information about its purpose or functionality.

Perhaps you could rename it to something more descriptive like `main_menu` which would better reflect what the method is doing.

Here's a quick example:

```ruby
def main_menu(my_class)
  # ... rest of your code
end

main_menu(my_class)
```

By doing so, you make it easier for others (and future you) to read and maintain the code. 

Best regards,
Temitope